src/arrayMethodJoin.js

```diff
@@ -4,8 +4,26 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(separator = ',') {
     // write code here
+    const join2 = [...this];
+    let result = '';
+
+    for (let i = 0; i < join2.length; i++) {
+      if (join2[i] === undefined || join2[i] === null) {
+        join2[i] = '';
+      }
+
+      if (i === 0) {
+        result += join2[i];
+      }
+
+      if (i > 0) {
+        result = result + separator + join2[i];
```

Comment:
What about the last iteration? You don't need a separator there

Reply:
I need separator at the last iteration, but make vithout it in first